service/src/intTest/java/uk/nhs/adaptors/gp2gp/EhrExtractStatusRepositoryTest.java

```diff
@@ -0,0 +1,44 @@
+package uk.nhs.adaptors.gp2gp;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.util.Optional;
+
+import uk.nhs.adaptors.gp2gp.configuration.MongoClientConfiguration;
+import uk.nhs.adaptors.gp2gp.extension.IntegrationTestsExtension;
+import uk.nhs.adaptors.gp2gp.model.EhrExtractStatus;
+import uk.nhs.adaptors.gp2gp.repository.EhrExtractStatusRepository;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+@ExtendWith({ SpringExtension.class, IntegrationTestsExtension.class })
+@SpringBootTest
```

Comment:
We'll use SpringBootTest here for now, but we need to put a story into the next sprint to remove it from integration tests when we move onto functional stories.

Reply:
Ok